src/main/generated/assets/draconicevolution/lang/en_us.json

```diff
@@ -145,10 +145,10 @@
   "gui.draconicevolution.dislocator.fuel_add_1.info": "Add 1 Ender Pearl from your inventory.\nEnder pearls are used as teleport fuel.",
   "gui.draconicevolution.dislocator.fuel_add_16.info": "Add 16 Ender Pearls from your inventory.\nEnder pearls are used as teleport fuel.",
   "gui.draconicevolution.dislocator.fuel_add_all.info": "Add all Ender Pearls from your inventory.\nEnder pearls are used as teleport fuel.",
-  "gui.draconicevolution.dislocator.mode_blink": "Use Mode:\nBlink",
-  "gui.draconicevolution.dislocator.mode_tp": "Use Mode:\nTeleport",
```

Comment:
This ![](https://ss.brandon3055.com/08aa3)
Was a GUI bug not a language issue. These line breaks are required.

Reply:
Ah, gotcha.  I'll remove that change.